### ISSUE TITLE
Provide the wheels for installation/distribution from within VS Code

### ISSUE THREAD

**Issue by DonJayamanne** [Microsoft/ptvsd#432]
Downstream issue https://github.com/Microsoft/vscode-python/issues/904

@brettcannon Please could you assign this issue to a milestone.

**Comment by zooba** [comment]
The wheels already include pre-built binaries for a few Python 3 versions on Windows. Can you be more specific about which extra platforms are required?

It's actually very difficult to provide prebuilt binaries for Linux (though manylinux should help cover most), and somewhat difficult for macOS, so perhaps VS Code should trigger its own build from sdist on those platforms?

**Comment by brettcannon** [comment]
A manylinux wheel is all that is needed here since there is no support on PyPI for any other Linux wheel type (and manylinux covers any Linux distro that uses glibc which is everything mainstream other than Alpine). As for macOS, I'm not sure where the difficulty is since you can build them with clang and so the only trick is building the wheels per Python version as necessary (i.e. it doesn't require Xcode or anything exotic).

**Comment by brettcannon** [comment]
Anyway, added this to the June 2018.2 milestone with the thinking that once the wheels are in place we can start worrying about automatic installation by VS Code in the first half of July.

**Comment by zooba** [comment]
Doesn't macOS tag by macOS version and require you to actually build on the same version (or earlier)? I was sure that was why Ned still builds [CPython releases](https://www.python.org/downloads/release/python-365/) on 10.6 (and now 10.9 as well)

**Comment by brettcannon** [comment]
I don't think we need to care about macOS 10.8 or older as [Mountain Lion is 5 years old](https://en.wikipedia.org/wiki/OS_X_Mountain_Lion). So supporting 10.9 and newer is already going to be more support than we will even have for the new analysis engine and will cover the vast majority of users anyway. So I think it's worth building a wheel for macOS, but there's no need to worry about supporting versions that you can't build for with the latest macOS release already.
